lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +17,34 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    if text == '':
+        return None
+    if all(symbol.isalpha() is False for symbol in text):
```

Comment:
`all`? Or `any`?

Reply:
Here I check option if all elements are punctuation. So, if all of them are symbols of punctuation, I return empty list. I don't know how I can use here 'any' instead of 'all'...

Reply:
Why do you need this code at all?